OpenRA.Mods.Common/HitShapes/Circle.cs

```diff
@@ -0,0 +1,40 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Drawing;
+using OpenRA.Graphics;
+
+namespace OpenRA.Mods.Common.HitShapes
+{
+	public class CircleShape : IHitShape
+	{
+		public WDist OuterRadius { get { return Radius; } }
+
+		[FieldLoader.Require]
+		public readonly WDist Radius = new WDist(426);
```

Comment:
@penev92: The internal default for health radius was also 426 ( = 10 pixel in RA/TD).